presto-release-tools/src/main/java/com/facebook/presto/release/git/GitCommands.java

```diff
@@ -28,18 +34,17 @@
         extends AbstractCommands
         implements Git
 {
+    private static final Splitter LINE_SPLITTER = Splitter.on("\n").trimResults().omitEmptyStrings();
```

Comment:
"\n" is probably the right thing to use, but should this use `System.lineSeparator()`?